### PR TITLE
#168842857 Fix responsiveness and resize blocks

### PR DESCRIPTION
**Pull request by mwafrika**
#### What does this PR do?
fix responsiveness and resize blocks
#### Description of Task to be completed?
- Increase the body width up to 100%,
- set Display to flex, Flex-direction to wrap, Justify-content: space-around.

#### How should this be manually tested?
when we use a different device other than the computer, The website will keep its responsiveness.

#### Any background context you want to provide?
Flex and media-queries properties have been used to resize the application

#### What are the relevant pivotal tracker stories?
#168842857

#### Screenshots (if appropriate)
Home page:

Screenshot (94)

User profile:
Screenshot (95)

View specific Article:
Screenshot (96)

### CODE REVIEW

File: UI/html_files/signup.html [hunk 1]
```diff
@@ -51,24 +51,24 @@ <h2><b>Motto:</b> Together we stand divided we fall</h2>
...
+          <div class="text1"><input type="password" id="passAdmin" placeholder="Password" value="" required></div>
+          <div class="text1"><input type="text" id="clients" placeholder="Role" required></div>
         </div>
         <div id="btn2"></div>
         <div id="btn">
           <!-- <button class="btn_post" onclick="checkClient()"> User</button> -->
-          <button class="btn_post"><a href="homepage.html"
-              style="color:white;font-size: 18px;">Login</a> </button>
+          <button class="btn_post" onclick= "window.location.href ='homepage.html';">
+            <p style="color:white;font-size: 16px;">Login</p>
```
Comment (Kasulejoseph) [inline UI/html_files/signup.html]: You need to get rid of inline styling... Have all the CSS in the CSS file.

**Review by Kasulejoseph — APPROVED** [review]

File: UI/html_files/signup.html (hunk 1 above)
Comment (mwafrika) [reply UI/html_files/signup.html]: already done, Please check!

File: index.html [hunk 2]
```diff
@@ -51,15 +49,15 @@ <h2><b>Motto:</b> Together we stand divided we fall</h2>
...
+          <div class="text1"><input type="password" id="passAdmin" placeholder="Password" value="" required></div>
         </div>
         <div id="btn2"></div>
         <div id="btn">
-          <!-- <button class="btn_post" onclick="checkClient()"> User</button> -->
-          <button class="btn_post"><a href="UI/html_files/view-all-post.html"
-              style="color:white;font-size: 18px;">Login</a>
-          </button>
+        <button class="btn_post" onclick="window.location.href ='UI/html_files/view-all-post.html';">
+       <p style="font-size:16px;">Login</p>
```
Comment (Kasulejoseph) [inline index.html]: You missed removing this...

**Review by Kasulejoseph — APPROVED** [review]
Looks good

**Pull request merged**